SU2_CFD/src/solvers/CIncEulerSolver.cpp

```diff
@@ -1870,7 +1904,7 @@ void CIncEulerSolver::SetPreconditioner(const CConfig *config, unsigned long iPo
 
     if (energy && !flame) Preconditioner[nDim+1][0] = Cp*Temperature/BetaInc2;
     else        Preconditioner[nDim+1][0] = 0.0;
-
+   
```

Comment:
Set up your IDE to clear trailing spaces

Reply:
👍